tests/policies/test_interactive.py

```diff
@@ -0,0 +1,53 @@
+"""Tests interactive policies."""
```

Comment:
This has been based very heavily on https://github.com/HumanCompatibleAI/imitation/pull/768 . How to acknowledge it properly?